master/server.go

```diff
@@ -16,11 +18,13 @@ type Job struct {
 func jobsHandler(w http.ResponseWriter, r *http.Request) {
 	var j Job
 	decoder := json.NewDecoder(r.Body)
-	decoder.Decode(&j)
+	if err := decoder.Decode(&j); err != nil {
+		return
+	}
 	fmt.Println(j)
 }
 
 func main() {
 	http.HandleFunc("/", jobsHandler)
-	http.ListenAndServe(":8080", nil)
+	http.ListenAndServe(lib.Port, nil)
```

Comment:
Port should be defined in the config, and have a default value of :8080 if not specified